osgp/platform/osgp-adapter-kafka-distributionautomation/src/main/java/org/opensmartgridplatform/adapter/kafka/da/infra/kafka/out/MeterReadingProducer.java

```diff
@@ -1,3 +1,10 @@
+/**
+ * Copyright 2020 Smart Society Services B.V.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ */
```

Comment:
Discussed this with @smvdheijden. We should actually remove this and some other classes, because they are left overs from the hackathon. That would also involve adapting some tests. To prevent scope screep, I will log a separate issue for removing the hackathon left overs.